jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -76,10 +76,9 @@ def getRepoURL() {
 }
 
 def getEnvironmentLogicalId() {
-	if (g_environment_logical_id == null) {
-		def token = g_login_token
+	if (g_environment_logical_id == null && service_config['domain'] != "jazz") {
```

Comment:
Why are we having to exclude jazz namespace?

Reply:
While deploying jazz services , the environment handlers and apis also login may not deployed.

Reply:
ahh, good point. We need to find another way to work around that. Lets come up with alternative ways that are better than the current "if" clauses.